src/lib/forms.js

```diff
@@ -12,6 +12,19 @@ const makeDropdown = (htmlFor, labelText, value, onChange, options, oValue, dVal
     </select>
   </label>
 );
+
+const makeYouthDropdown = (htmlFor, labelText, value, onChange, options, oValue, dValue) => (
+  <label htmlFor={htmlFor} style={{ paddingTop: '12px' }} id={htmlFor}>
+    {labelText}
+    <br />
+    <select id={htmlFor} value={value} onChange={(event) => onChange(event, 'youthPicsId')}>
+      <option id="blank-option" key="blank-option" value="">---</option>
```

Comment:
@Unesdala you just need to use the same function and use the htmlFor to be the state variable, then replace it in the onChange where the state variable is there in single quotes right now, please fix for both delete dropdowns and remove the second function that is almost a complete identical one